collect_app/src/main/java/org/odk/collect/android/utilities/InstanceUploaderUtils.java

```diff
@@ -154,7 +154,7 @@ public static boolean formShouldBeAutoSent(FormsRepository formsRepository, Stri
      *
      * If the form explicitly sets the auto-delete property, then it overrides the preference.
      */
-    public static boolean formShouldBeAutoDeleted(FormsRepository formsRepository, String jrFormId, String jrFormVersion, boolean isAutoDeleteAppSettingEnabled) {
+    public static boolean shouldFormBeDeleted(FormsRepository formsRepository, String jrFormId, String jrFormVersion, boolean isAutoDeleteAppSettingEnabled) {
```

Comment:
Might be nice to have these static methods in a new `InstanceUtils` class in either `instancemanagement` or `instances`. I can see us pulling the other methods out at some point (maybe as part of #3988).